### PR TITLE
Programmes 6610 fix twitter embeddability

### PR DESCRIPTION
**Pull request by BjornTwachtmann**

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Controller/AncillaryPages/PlayerController.php [hunk 1]
```diff
@@ -23,6 +23,8 @@ public function __invoke(
         VersionsService $versionsService,
         Clip $clip
     ) {
+        // This frame needs to display off the BBC site
+        $this->response()->headers->remove('X-Frame-Options');
```
Comment (Tasiobg) [inline src/Controller/AncillaryPages/PlayerController.php]: What do you think about only removing this header when `$clip->isExternallyEmbeddable()` is true?

Comment (BjornTwachtmann) [reply src/Controller/AncillaryPages/PlayerController.php]: This page is only shown when isExternallyEmbeddable is true

Comment (BjornTwachtmann) [reply src/Controller/AncillaryPages/PlayerController.php]: it's 404 otherwise

**Pull request merged**